src/apps/actions.js

```diff
@@ -0,0 +1,25 @@
+export const NEW_BUNDLE = 'NEW_BUNDLE';
+export const NEW_ERROR = 'NEW_ERROR';
+export const FETCH_BUNDLE_ERROR = 'FETCH_BUNDLE_ERROR';
+
+export function newBundle(bundle, inputs) {
+    return {
+        type: NEW_BUNDLE,
```

Comment:
Thoughts on following [FSA](https://github.com/acdlite/flux-standard-action) (i.e. sticking stuff in `payload`)? I started following it in some repo somewhere. Not saying we have to switch to it but just wanted to bring it up in case you think its a good idea/haven't seen it.